test/unit/meson.build

```diff
@@ -94,7 +101,8 @@ if get_option('enable_tests')
 
   foreach test : tests
     exe = executable('test_@0@'.format(test), 'test_@0@.c'.format(test),
-      include_directories: [platform_inc],
+      # '.' to find autogenerated "test_config.h"
```

Comment:
I think the addition of `.` is not necessary, as it is automatically added by meson AFAIR.

Reply:
I added `.` as the build did not work as is:

```
FAILED: test/unit/test_rzpipe.p/test_rzpipe.c.o
ccache clang -Itest/unit/test_rzpipe.p -I. -I.. -I../librz/include -Isubprojects/sdb -I../subprojects/sdb -Isubprojects/sdb/src -I../subprojects/sdb/src -I../shlr/spp -Ishlr -I../shlr -I../librz/asm/arch/include -I../librz/asm/arch -I../librz/asm/arch/h8300 -I../librz/asm/arch/hexagon -I../librz/asm/arch/msp430 -I../librz/asm/arch/rsp -I../librz/asm/arch/mcore -I../librz/asm/arch/v850 -I../librz/asm/arch/propeller -I../librz/asm/arch/ebc -I../librz/asm/arch/cr16 -I../librz/asm/arch/8051 -I../librz/asm/arch/v810 -I../librz/asm/arch/or1k -I../librz/analysis/arch/gb -I../librz/analysis/arch/hexagon -I../librz/bin/format/elf -I../librz/bin/mangling -I../librz/bin/format -I../shlr/heap/include -Xclang -fcolor-diagnostics -pipe -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -O2 -g --std=gnu99 -D_GNU_SOURCE -Werror=sizeof-pointer-memaccess -fvisibility=hidden -MD -MQ test/unit/test_rzpipe.p/test_rzpipe.c.o -MF test/unit/test_rzpipe.p/test_rzpipe.c.o.d -o test/unit/test_rzpipe.p/test_rzpipe.c.o -c ../test/unit/test_rzpipe.c
../test/unit/test_rzpipe.c:7:10: fatal error: 'test_config.h' file not found
#include "test_config.h"
         ^~~~~~~~~~~~~~~
```

This target uses `implicit_include_directories: false` a few lines below. We can either remove that or add `.`. Which one is preferable?

Reply:
implicit_include_directories: false is necessary